setup.py

```diff
@@ -115,14 +117,15 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     install_requires=[
+        "cython~=3.0",
```

Comment:
Cython is not a runtime dependency, so doesn't need to be in `install_requires`. It's only needed for building.

Reply:
Should I then remove it from `requirements.txt` too? Or is it useful to have it there when someone wants to install both build and runtime dependencies using `pip install -r requirements.txt`?

Reply:
`requirements.txt` is for developers to set up a useful working environment, so the Cython dependency should remain in there. The `install_requires` section in `setup.py` should only specify the packages required to use an already-built wheel after it has been installed (it's not a great keyword name IMO).

Reply:
Thanks for the explanation. It's done.